aggregators/median-aggregator.ts

```diff
@@ -33,16 +29,16 @@ const medianAggregator: Aggregator = {
       }
 
       return {
-        ...price,
+        ...price,//czy ceny, które nie zostały uwzględnione przy liczeniu mediany, nie powinny stąd wylecieć?
```

Comment:
That's a good question. I think both options have their advantages. @jakub-wojciechowski what do you think?

Reply:
Leaving as is for now...